### ISSUE TITLE
launchDashboardWithListLeaderboardsPage function missing

### ISSUE THREAD

**Issue by stanokudlac** [marmalade/openfeint#3]
Hi, I would like to display OF in the page where the user can see a list of all leaderboards however it is possible to display only a specific leaderboard. Could you please add this functionality when I call s3eNOFlaunchDashboardWithHighscorePage with NULL parameter or create a new function something like s3eNOFlaunchDashboardWithListHighscoresPage?

Thank you for great extension!

**Comment by faizann** [comment]
Hi 
I think this is the OpenFeint API launchDashboardWithListLeaderboardsPage that needs to be called. 
I will add this to extension and update code in 2 days.

**Comment by stanokudlac** [comment]
Hi

It would be nice if you could also provide an update to the latest version 
of OpenFeint because Apple is currently rejecting games using pre 2.12.5 
because of some non-public API functions used in OpenFeint code as stated 
here http://support.openfeint.com/dev/updatebadge/.

I tried to build your extension with the latest version of OF by myself. 
However I am not familiar with XCode and Mac's at all and probably I am 
missing something here. I was able to build it successfully but my library 
file has only 166KB and yours has 41,6MB. I can't get Marmalade's deploy 
tool to link to OF. I tried to use "-framework OpenFeint" in Additional Link 
Options as well but it crashes the linker totally. Note that I am using 
Marmalade on Windows using Visual Studio 2008. If you could point me how to 
properly set up the project in XCode for your extension I will be very 
thankful.

Thanks
Stano

---

Od: "Faizan Naqvi" 
reply@reply.github.com
Dátum: 14. novembra 2011 18:49
Komu: "stanokudlac" stano.kudlac@gmail.com
Predmet: Re: [openfeint] launchDashboardWithListLeaderboardsPage function 
missing (#3)

> Hi
> I think this is the OpenFeint API launchDashboardWithListLeaderboardsPage 
> that needs to be called.
> I will add this to extension and update code in 2 days.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/marmalade/openfeint/issues/3#issuecomment-2733811
> 
> __________ Information from ESET NOD32 Antivirus, version of virus 
> signature database 6629 (20111114) __________
> 
> The message was checked by ESET NOD32 Antivirus.
> 
> http://www.eset.com

__________ Information from ESET NOD32 Antivirus, version of virus signature database 6634 (20111116) __________

The message was checked by ESET NOD32 Antivirus.

http://www.eset.com

**Comment by faizann** [comment]
Hi

I just updated code for OpenFeint extension.
I have added launchDashboardWithListLeaderboardsPage as **s3eNOFlaunchDashboardWithListLeaderboardsPage**

As to the other **updateBadge** issue in OpenFeint code, I have commented it out in OpenFeint itself as per the solution provided at http://support.openfeint.com/dev/updatebadge/#changeItMyself

Have a look at diff file below. 
https://github.com/marmalade/openfeint/commit/5e46206028f2eb720612dfb79eb5a79b66cd6915#diff-2

The new version of OpenFeint is 2.12 while the one in our code is 2.10. I will try with new SDK in coming few days.
